### DISCUSSION TITLE
Getting output files off AWS environ

### DISCUSSION THREAD

**Discussion by fortworthjack** [dtcenter/METplus#1600] [File I/O] [answered]
I was walking through the MODE tutorial material after the last week's session, and was happy to be able to produce output .ps files and view them using the "gv" command!

My question relates to an earlier post #1295?, in which I was curious if there is any way to get any of the output data OFF/OUT the AWS environment, and to my local machine (I'm ssh-ing from a Windows OS to AWS).  Like, if I wanted to save images, a PDF, or raw data CSV or netCDF created from within AWS.  Or, is it just too closed an environment for such options to work easily?  @fisherhucar 

![image](https://user-images.githubusercontent.com/6979158/166319829-71db5e42-4e57-49f6-ad72-011fdbe3496c.png)

**Comment by hankenstein2** [comment, marked as answer]
In general if you can ssh into the AWS you can get data out. Depending on what software you are using on your windows machine there should be a way to pull the data to your windows machine. I.e. you can't do anything while you are on the AWS EC2 instance but you should be able to initiate a data pull on your windows box.

I use mobaxterm on windows and they have a handy windows like icon tool box in a column on the left side of the window that you can do a file list and click on the file to download to your windows machine.

putty is different and if you use something else it too will be different but it should be possible.

**Reply by fortworthjack** [reply]
Thanks for the reply (I had checked earlier for a reply, but didn't see this; my mistake).  It's nice to see the options, and I can try that the next time I have an EC2 setup for METplus from my Windows/Putty environment.

**Comment by hankenstein2** [comment]
Hi Jack, were you able to get files off the EC2 instance or would you like more suggestions?
